Makefile

```diff
@@ -1,12 +1,14 @@
-generate: schemas readme template fields
+VERSION ?= 1.0.0-beta1
```

Comment:
@andrewkroh Should be perhaps introduce a `.version` file in the repo where all tools can read from?